app/controllers/v1/released_controller.rb

```diff
@@ -20,8 +20,8 @@ def update
 
       # add the release tags, and reuse tags if already associated with this PURL
       purl.refresh_release_tags(actions)
+      UpdatePurlMetadataService.new(nil, purl).write!(only: [:meta])
```

Comment:
Can we move the meta writing into its own class? I think that makes it simpler than having a bunch of conditionals around the `only:` parameter.  Please restore the indexer message here, or create a `ReleaseService` which combines writing the meta.json + kafka message.

Reply:
It seems like you have a very clear notion of how you'd like this work done, and I'm not sure I share that clarity. I'll just close this PR and you can pick up the underyling ticket.